content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+brand new DVC 1.0 is ready for public beta testing.
+
+The new DVC is inspired by discussions and contributions from our community -
+both fresh ideas and bug reports 😅.
+
+Here are the most significant features we’re excited to be rolling out soon:
+
+### [Run cache](https://github.com/iterative/dvc/issues/1234)
+
+_Learnings:_ Forcing users to make Git commits for each ML experiment creates
+too much overhead.
+
+DVC 1.0 has a "long memory" of DVC commands runs. This means it can identify if
+a `dvc repro` has already been run and save compute time by returning the cached
+result - _even if you didn't Git commit that past run_.
+
+We added the run-cache with CI/CD systems and other MLOps and DataOps automation
+tools in mind. No more auto-commits needed after `dvc repro` in the CI/CD system
+side.
+
+### [Multi-stage DVC files](https://github.com/iterative/dvc/issues/1871)
+
+_Learnings:_ ML pipelines evolve much faster than data engineering pipelines.
+
+We redesigned the DVC-metafile format to make saved pipelines more interpretable
+and editable. Pipeline stages are now saved in a single metafile, with all
+stages stored together instead of in separate files.
+
+Another significan step was the removed checksums from the pipeline metafile.
+This improves its human-readability.
+
+### [Plots](https://github.com/iterative/dvc/issues/3409)
+
+_Learnings:_ Versioning metrics and plots are no less important than data
+versioning.
+
+Countless users asked us when we'd support metrics visualizations. Now it's
+here: DVC 1.0 introduces a metrics file visualization command,
+`dvc metrics diff`. DVC plots are powered by the
+[Vega-Lite](https://vega.github.io/vega-lite/) graphic library.
+
+This function is designed not only for showing visualizations based on the
+current state of your project, but it can also combine multiple plots from your
+Git history in a single chart so you can compare results across commits. Users
+can visualize how, for example, their model accuracy in the latest commit
+differs from another commit (or even multiple commits).
+
+```dvc
+$ dvc plot diff -d logs.csv HEAD HEAD^ d1e4d848 baseline_march
+file:///Users/dmitry/src/plot/logs.html
+$ open logs.html
+```
+
+![](/uploads/images/2020-05-04/dvc-plot.png)
+
+```dvc
+$ dvc plot diff -d logs.csv HEAD HEAD^ d1e4d848 baseline_march \
+        -x loss --template scatter
+file:///Users/dmitry/src/plot/logs.html
+$ open logs.html
+```
+
+![](/uploads/images/2020-05-04/dvc-plot-scatter.png)
+
+### [Data transfer optimizations](https://github.com/iterative/dvc/issues/3488)
+
+_Learnings:_ In ML projects, data transfer optimization is still the king.
+
+We've done substantial work on optimizing data management commands, such as
+`dvc pull \ push \ status -c \ gc -c`. Now, based on the amount of data, DVC can
+choose an optimal data remote traversing strategy.
+
+[Wini-indexes](https://github.com/iterative/dvc/issues/2147) were introduced to
```

Comment:
better to put link to @pmrowla report

Reply:
Provided the link and the plot. Looks very solid 😎